boring/src/x509/verify.rs

```diff
@@ -77,19 +102,43 @@ impl X509VerifyParamRef {
         }
     }
 
+    /// Gets verification flags.
+    #[corresponds(X509_VERIFY_PARAM_get_flags)]
+    pub fn flags(&mut self) -> X509VerifyFlags {
```

Comment:
Why is this `&mut self`?

Reply:
That's what openssl-rust has, and bindgen generates `pub fn X509_VERIFY_PARAM_get_flags(param: *mut X509_VERIFY_PARAM) -> ::std::os::raw::c_ulong;` with a `mut` pointer.